cmd/kube-scheduler/app/server.go

```diff
@@ -113,7 +119,7 @@ func runCommand(cmd *cobra.Command, opts *options.Options, registryOptions ...Op
 
 	// Activate logging as soon as possible, after that
 	// show flags with the final logging configuration.
-	if err := opts.Logs.ValidateAndApply(); err != nil {
+	if err := opts.Logs.ValidateAndApply(utilfeature.DefaultFeatureGate); err != nil {
```

Comment:
this will work as long as scheduler enablement of the feature is only done via a `--feature-gates` flag. I don't see a field in the scheduler config for enabling feature gates at the moment, so that seems workable.